Add custom label for runner (#32)
cybozu-go/github-actions-controller: docs/design.md

is a label to route jobs to specific types of runners. Users usually use
self-hosted runners by setting `self-hosted` to `runs-on` in a workflow
definition. If custom labels are given, users are allowed to set one of the
custom label value to `runs-on`. This is useful when you want to use multiple
types of runners, for example, `highmem`and `highcpu`.

> In github-actions-controller, The `RunnerPool` name (namespaced name) is set as the custom label.

Author: ```suggestion
github-actions-controller sets the namespaced name of `RunnerPool` as a custom label.
```
